hack/start-agent.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# arguments: TARGET, YGGDRASIL_PATH, (optional) DEVICE_WORKER_PATH
```

Comment:
optional: Named arguments or using flags will be more descriptive and the parsing is more precise than positional approach, e.g. https://github.com/project-flotta/flotta-scale-tests/blob/main/scripts/run_test_plan.sh#L3

Reply:
done